storage.rb

```diff
@@ -0,0 +1,87 @@
+module Storage
+  def save_data
+    save_books
+    save_people
+    save_rentals
+  end
+
+  def save_books
+    file_path = File.join('data', 'books.json')
+    File.write(file_path, JSON.generate(books.map(&:to_h)))
+  rescue StandardError => e
+    puts "An error occurred while saving books: #{e.message}"
+  end
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍